src/WorkItems/WorkItems.csproj

```diff
@@ -28,10 +28,11 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.2" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.13.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.2" />
-    <PackageReference Include="Microsoft.Json.Schema" Version="1.1.4" />
-    <PackageReference Include="Microsoft.Json.Schema.Validation" Version="1.1.4" />
```

Comment:
><PackageReference Include="Microsoft.Json.Schema.Validation" Version="1.1.4" /> [](http://example.com/codeflow?start=4&length=79)

FYI I also see this in solution, not sure if we want to update it as well. It is used in the generation of c# files.

`<package id="Microsoft.Json.Schema.ToDotNet" version="1.1.4"`